application/common/models/Reset.php

```diff
@@ -146,24 +146,22 @@ public function send()
 
     public function sendPrimary()
     {
-        /**
-         * @todo send email to $user->email with reset code
```

Comment:
I created github issues instead and removed @todo to clean up scrutinizer reports. I left a couple @todo in places where there was code but something needed to be finished and removed it from places where there was no code at all and would not be confused as done.